### ISSUE TITLE
Ensure color matrix is accessible to screenreaders

### ISSUE THREAD

**Issue by jenniferthibault** [18F/brand#51]
Make sure that the color matrix and text is readable by screen readers, and complies with other accessibility standards. 

![18f-color-palette](https://cloud.githubusercontent.com/assets/11636908/12268003/ec7e3e02-b918-11e5-858c-169f779cdce7.png)

**Comment by toolness** [comment]
Do you mean that the color matrix should be converted to HTML, so that someone could read it with a screen reader? Or something else?

Coincidentally, I'm working on #64 and making an HTML version of this color matrix is very doable with the CSS I've written so far. Could also help with #85!

**Comment by jenniferthibault** [comment]
Thanks @toolness !! To capture what we talked about in person here, I meant either getting the text out of the image and into HTML, or adding the appropriate tags to be comprehensible to screen readers. 

Love your approach of building the whole shebang in HTML, which will make this element easily reproducible, and get a start on making it responsive. Do you want us to make a mobile mockup for this, or do you already have ideas?

**Comment by jenniferthibault** [comment]
Transferring assignment over to @rtwell, who was interested in thinking through mobile layout for the matrix with you @toolness .

& @ericronne tapping in for backup review.

**Comment by toolness** [comment]
Sounds good @jenniferthibault, looking forward to working on the mobile version with you @rtwell!

**Comment by rtwell** [comment]
here's my first pass, with gentle nudges from @ericronne. 
cc @jenniferthibault @thisisdano @amberwreed @toolness 
## Default view
![18f-brand-mobile_collapse](https://cloud.githubusercontent.com/assets/12564977/15726156/fc661892-2814-11e6-9f6b-b51a6dea9f3c.png)
## First color expanded
![18f-brand-mobile_expand](https://cloud.githubusercontent.com/assets/12564977/15726166/084b0154-2815-11e6-8d97-86eb5c30dd44.png)
## All expanded
![18f-brand-mobile_expand-all](https://cloud.githubusercontent.com/assets/12564977/15726208/33943ee8-2815-11e6-85b1-bb671864f565.png)

**Comment by toolness** [comment]
Ooooo NICE! :100: 

I like this. Lemme think how best to implement it and I'll attempt to code it up.

Regarding the base color palette (not the combinations)--the existing stylesheet actually does that in a nice responsive way very similar to the one in your mockup above, only it's two color swatches per row instead of three:

![2016-06-01_17-37-16](https://cloud.githubusercontent.com/assets/124687/15726574/9a5d85e4-281f-11e6-9098-d9a5c0424bb6.png)

Is it ok to keep the existing layout or would you prefer 3 swatches per row?

**Comment by toolness** [comment]
Ok, so I'm actually getting to be a bit tight on time (I'm starting a new full-time project on Monday), and while your mockups are *really* awesome, I've implemented something that falls rather short of them, yet are still inspired by them, in #94. I'd still like to implement your original mockups at a later point (someone else can do it too of course) but in the interests of time I'm hoping we can get away with something a bit less cool. Curious to know your thoughts!
